erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -27,9 +27,19 @@ def validate(self):
 
 	def before_save(self):
 		self.set_item_locations()
+		# set percentage picked in SO
+		for i in self.get('locations'):
+			if i.sales_order and frappe.db.get_value("Sales Order",i.sales_order,"per_picked") > 0:
+				frappe.throw("Row " + str(i.idx) + " has been picked already!")
 
 	def before_submit(self):
 		for item in self.locations:
+			# set picked_qty to stock_qty, before submit
+			item.picked_qty = item.stock_qty
```

Comment:
if item.picked_qty is set by the user manually then shouldn't be override

Reply:
Handled.